script.js

```
@@ -277,6 +365,39 @@ document.addEventListener('DOMContentLoaded', function() {
+        const settings = JSON.parse(savedSettings);
+        if (settings.profilePhoto) photoPreview.src = settings.profilePhoto;
+      } catch (e) {}
+    }
+    if (uploadBtn && photoInput && photoPreview) {
+      uploadBtn.addEventListener('click', function() {
+        photoInput.click();
+      });
+      photoInput.addEventListener('change', function(e) {
+        const file = e.target.files[0];
+        if (file && file.type.startsWith('image/')) {
+          const reader = new FileReader();
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)